webdriver-spec.html

```diff
@@ -4108,11 +4108,16 @@
  <li><p>Otherwise return false.
 </ol>
 
-<p>An <a>element</a> is
- <dfn data-lt="scroll into view|scrolls into view">scrolled into view</dfn>
- by calling <a>scrollIntoView</a>
- with the <a><code>ScrollIntoViewOptions</code></a>
- <code>{behavior: "instant", block: "end", inline: "nearest"}</code>.
+<p>When required to
+<dfn data-lt="scrolled into view|scrolls into view">scroll into view</dfn>
+<a>element</a> <var>element</var>,the <a>endpoint node</a> steps are:
```

Comment:
You could just do `<var><a>element</a></var>` here so the language flows better. You can also drop “the endpoint node steps are”: it is always called from a driver implementation.

I also liked the language from before better. To me, “an element is scrolled into view” sounds better than “when required to scroll into view element element”.